warehouses/facility_ops/ingest/beamlines/moderator_performance/moderator_performance.py

```
@@ -0,0 +1,210 @@
+        row_filter=EqualTo(c_beamline, beamline),
+        selected_fields=(c_run_number, c_cycle),
+    ).to_arrow()
+    if beamline_peaks.num_rows > 0:
+        cycle_groups = beamline_peaks.group_by(c_cycle).aggregate(
+            [(c_run_number, "list")]
+        )
+        fitted_runs = dict(
+            zip(
+                cycle_groups[c_cycle].to_pylist(),
+                sorted(cycle_groups[c_run_number + "_list"].to_pylist()),
+            )
```

Ruff (0.15.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)